modules/secrets.js

```diff
@@ -47,7 +47,7 @@ const JWT_SIGNING_TOKEN =
 /**
  * Sentry DSN for error reporting
  */
-const SENTRY_DSN = getSecret('sentry.dsn');
+const SENTRY_DSN = getSecret('sentry.publicDsn');
```

Comment:
Sentry updated their requirements a while ago so that all SDKs just use the public DSN, no more private DSN needed. I've added a new value to parameter store and we can remove the old one once this change is out.